06-statistical-testing.Rmd

```diff
@@ -67,39 +67,39 @@ recs_des <- recs_2020 %>%
 
-The functions in the {survey} package allow for the correct estimation of the variances. This chapter will cover the following statistical tests with survey data and functions:
+The functions in the {survey} package allow for the correct estimation of the variances. This chapter covers the following statistical tests with survey data and functions:
 
-* Comparison of proportions `svyttest()`
-* Comparison of means `svyttest()`
-* Goodness of fit tests `svygofchisq()`
-* Tests of independence `svychisq()`
-* Tests of homogeneity `svychisq()`
+* Comparison of proportions (`svyttest()`)
+* Comparison of means (`svyttest()`)
+* Goodness of fit tests (`svygofchisq()`)
+* Tests of independence (`svychisq()`)
+* Tests of homogeneity (`svychisq()`)
 
 ## Dot Notation {#dot-notation}
 
-Up to this point, we have shown functions that use wrappers from the {srvyr} package. This means that the functions work with tidyverse syntax. However, the functions in this chapter do not have wrappers in the {srvyr} package and are instead used directly from the {survey} package. Therefore, the design object is *not* the first argument, and to use these functions with the magrittr pipe (`%>%`) and tidyverse syntax, we will need to use dot (`.`) notation^[This could change in the future if another package is built or {srvyr} is expanded to work with **tidymodels** but no such plans are known at this time.]
+Up to this point, we have shown functions that use wrappers from the {srvyr} package. This means that the functions work with tidyverse syntax. However, the functions in this chapter do not have wrappers in the {srvyr} package and are instead used directly from the {survey} package. Therefore, the design object is *not* the first argument, and to use these functions with the magrittr pipe (`%>%`) and tidyverse syntax, we need to use dot (`.`) notation.^[This could change in the future if another package is built or {srvyr} is expanded to work with **tidymodels** but no such plans are known at this time.]
 
-Functions that work with the magrittr pipe (`%>%`) have the data as the first argument. When we run a function with the pipe, it automatically places anything to the left of the pipe into the first argument of the function to the right of the pipe. For example, if we wanted to take the `mtcars` data and filter to cars with six cylinders, we can write the code in at least four different ways:
+Functions that work with the magrittr pipe (`%>%`) have the dataset as the first argument. When we run a function with the pipe, it automatically places anything to the left of the pipe into the first argument of the function to the right of the pipe. For example, if we wanted to take the `mtcars` data and filter to cars with six cylinders, we can write the code in at least four different ways:
```

Comment:
Should we change this example to towny to match ch4?

Reply:
Agreed and updated example.